services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/storage.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict, List, Optional
+
+from .settings import ServiceSidecarSettings
+from .utils import assemble_container_names, validate_compose_spec
+
+
+class SharedStore:
```

Comment:
it is a dataclass? or a pydantic class no? this would remove some boilerplate code here. 
Also if possible I would change either the file name to follow the object in it or the object. storage being already a service it might be a bit confusing

Reply:
I would rename the module to `shared_store`. Will avoid migrating to something like dataclasses or pydantic as I do not find a clear advantage for this use case.

Reply:
changed my mind, refactored to Pydantic model.